src/components/Cat.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from "react"
+import {connect} from "react-redux"
+import {removeCat} from "./../action_creators"
+
+function Cat ({src, fact, id, dispatch}) { // todo fix lint: react/jsx-no-bind
+  return (<div className="cat" onClick={() => dispatch(removeCat(id))}>
```

Comment:
Eslint was complaining about using bind or arrow functions in props because they get instantiated every render. That suggests two alternatives:
1. pass in from higher level (pushing the problem up and mucking with separation of concerns)
2. write a function to generate the right function signature. Sounds like `bindactioncreators` might be the way to go?